{"name": "test_DB_link", "status": "broken", "statusDetails": {"message": "NameError: name 'driver' is not defined", "trace": "@allure.story('確認驗證碼能是否正常/並且確認')\n    @pytest.mark.run(order=7)\n    def test_DB_link():\n     time.sleep(5)\n     conn = pymysql.connect(**db_setting)\n     with conn.cursor() as cursor:\n    \n        command =\"select code From telligent_member.member_captcha ORDER BY expired_time desc LIMIT 1;\"\n        cursor.execute(command)\n        result =cursor.fetchone()\n        print(result)\n>    WebDriverWait(driver, 10).until(EC.presence_of_element_located((By.XPATH, \"/html/body/div[1]/div/div/main/div/div[2]/div/div/div/form/div/label/div/div[1]/div/input\")))\nE    NameError: name 'driver' is not defined\n\nSSO_Line_test.py:91: NameError"}, "attachments": [{"name": "stdout", "source": "7295631f-3d92-47f0-9db3-b25a5e0ba4bc-attachment.txt", "type": "text/plain"}], "start": 1669948640268, "stop": 1669948645275, "uuid": "bcf01228-92df-4abe-9a66-f20017977ecd", "historyId": "85528444a32638a2e3664690277399f4", "testCaseId": "eb2c479fdd59ca25da8d47312f8f3f94", "fullName": "SSO_Line_test#test_DB_link", "labels": [{"name": "story", "value": "確認驗證碼能是否正常/並且確認"}, {"name": "tag", "value": "run(order=7)"}, {"name": "suite", "value": "SSO_Line_test"}, {"name": "host", "value": "C010700042"}, {"name": "thread", "value": "16756-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "SSO_Line_test"}]}